### DISCUSSION TITLE
Puntuación Parte 1 incompleta

### DISCUSSION THREAD

**Discussion by bozzimuc** [IIC2133-PUC/2024-2#109] [Tarea 1] [answered]
Buenas tardes. No he logrado hacer el Evento LEAVE de la Parte 1 de la Tarea 1 en complejidad logarítmica (creo que lo tengo en complejidad lineal), pero sí el resto de Eventos de la Parte 1. Respecto a eso, tengo dos preguntas:
1. ¿Podré recibir puntaje de los otros Eventos en caso de que LEAVE no cumpla con complejidad pedida?
2. ¿Recibiré puntaje parcial en caso de que LEAVE dé el resultado correcto pero sin la complejidad pedida?
Muchas gracias

**Comment by tomastrivino** [comment, marked as answer]
Los tests son independientes entre sí. Esto significa que podrías tener puntaje en un test de ENTER (1/1) y no en uno de LEAVE (0/1) <Nota: generalmente los tests involucran varios eventos, es decir, no se puede testear el LEAVE si no hay ENTER en el test mismo>. El puntaje en los otros test dependerá de que cumplas con el output solicitado, el cual **sí podría ser influenciado por eventos que ocurran dentro de cada uno**.

Si no cumples con la complejidad, no se espera que pasen los tests.
Saludos
